tests/integration/base_deploy_test.go

```diff
@@ -107,6 +107,13 @@ func StartTestCluster(t func() *testing.T, namespace, storeType string, helmInst
 	i := installer.NewCephInstaller(t, kh.Clientset, rookVersion)
 
 	op := &TestCluster{i, kh, nil, t, namespace, storeType, helmInstalled, useDevices, mons}
+
+	if rookVersion != installer.VersionMaster {
+		// make sure we have the images from a previous release locally so the test doesn't hit a timeout
+		assert.NoError(t(), kh.GetDockerImage("rook/ceph:"+rookVersion))
```

Comment:
does any output go to the test logs that shows progress of these large image pulls? or do we need to add any start/finish logging around them?

Reply:
Yes, it does show the output from docker pull so you can see progress. For an example, see the early output of the `TestUpgradeSuite` [here](https://jenkins.rook.io/blue/rest/organizations/jenkins/pipelines/rook/pipelines/rook/branches/PR-2029/runs/8/nodes/48/steps/103/log/?start=0).